python/grass/temporal/temporal_algebra.py

```diff
@@ -2275,11 +2275,9 @@ def eval_condition_list(self, maplist, inverse: bool = False):
              - [True,  '&&', False]              -> False
              - [False, '||', True]               -> True
 
-        :param tvarexpr: List of GlobalTemporalVar objects and map lists.
-                     The list is constructed by the TemporalAlgebraParser
-                     in order of expression evaluation in the parser.
-
-        :return: Map list with conditional values for all temporal expressions.
+        :param maplist: A list of maps
+        :param inverse:
+        :return: Map list with conditional values evaluated
```

Comment:
This whole function's parameters and description were unrelated and copied from another function. I wrote a real minimum, but didn't know what to write for "inverse".

Feel free to suggest anything for these three lines and commit it right away, if it is already better than what will be here otherwise. Wrong docs are kinda worse than no docs in my opinion.